test/Microsoft.ML.Functional.Tests/Datasets/FeatureColumn.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.ML.Functional.Tests.Datasets
+{
+    /// <summary>
+    /// A class to hold a feature column.
+    /// </summary>
+    internal sealed class FeatureColumn
+    {
+        public float[] Features { get; set; }
+    }
```

Comment:
maybe move it to the other file, the Iris.

Reply:
This is designed to be generic for tests against any features columns, not just the iris, so I'll keep it in its own file for now, similar to `FeatureContributionOutput'.

---
In reply to: [261830507](https://github.com/dotnet/machinelearning/pull/2803#discussion_r261830507) [](ancestors = 261830507)